### DISCUSSION TITLE
Unable to deploy Argo latest version

### DISCUSSION THREAD

**Discussion by arunanandhan** [argoproj/argo-helm#1703] [Q&A]
I am getting the error 

`│ Error: template: argo-cd/templates/argocd-configs/argocd-cm.yaml:15:6: executing "argo-cd/templates/argocd-configs/argocd-cm.yaml" at <include "argo-cd.config.cm" .>: error calling include: template: argo-cd/templates/_helpers.tpl:187:49: executing "argo-cd.config.cm" at <$config>: wrong type for value; expected map[string]interface ***; got string`

 when deploying argo latest version 5.16.2, Am i missing something in values file? 
 
 @pdrastil

**Comment by pdrastil** [comment]
Hi @arunanandhan, based on the error you are passing invalid value into `configs.cm` section. YAML is strongly typed and error is telling you it's expecting key value pairs where key is a string. But section contains just a string.
